app/models/chargeback_container_project.rb

```diff
@@ -0,0 +1,153 @@
+class ChargebackContainerProject < Chargeback
+  set_columns_hash(
+    :start_date               => :datetime,
+    :end_date                 => :datetime,
+    :interval_name            => :string,
+    :display_range            => :string,
+    :entity_name              => :string,
+    :cpu_allocated_metric     => :float,
+    :cpu_allocated_cost       => :float,
+    :cpu_used_cost            => :float,
+    :cpu_used_metric          => :float,
+    :cpu_cost                 => :float,
+    :cpu_metric               => :float,
+    :fixed_compute_1_cost     => :float,
+    :fixed_compute_2_cost     => :float,
+    :fixed_2_cost             => :float,
+    :fixed_cost               => :float,
+    :memory_used_cost         => :float,
+    :memory_used_metric       => :float,
+    :memory_cost              => :float,
+    :memory_metric            => :float,
+    :memory_allocated_cost    => :float,
+    :memory_allocated_metric  => :float,
+    :total_cost               => :float
+  )
+
+  def self.report_col_options
+    {
+      "cpu_allocated_cost"       => {:grouping => [:total]},
+      "cpu_allocated_metric"     => {:grouping => [:total]},
+      "cpu_cost"                 => {:grouping => [:total]},
+      "cpu_metric"               => {:grouping => [:total]},
+      "cpu_used_cost"            => {:grouping => [:total]},
+      "cpu_used_metric"          => {:grouping => [:total]},
+      "fixed_compute_1_cost"     => {:grouping => [:total]},
+      "fixed_compute_2_cost"     => {:grouping => [:total]},
+      "fixed_cost"               => {:grouping => [:total]},
+      "fixed_storage_1_cost"     => {:grouping => [:total]},
+      "fixed_storage_2_cost"     => {:grouping => [:total]},
+      "memory_cost"              => {:grouping => [:total]},
+      "memory_metric"            => {:grouping => [:total]},
+      "memory_used_cost"         => {:grouping => [:total]},
+      "memory_used_metric"       => {:grouping => [:total]},
+      "memory_allocated_cost"    => {:grouping => [:total]},
+      "memory_allocated_metric"  => {:grouping => [:total]},
+      "total_cost"               => {:grouping => [:total]}
+    }
+  end
+
+  def self.build_results_for_report_ChargebackContainerProject(options)
```

Comment:
@zeari it seems that there's a lot of code duplication with `build_results_for_report_ChargebackVm`. I know it's hard to refactor to improve code reuse but right now there's more than 50% of the code that is just copy/paste (too much).